packages/core/src/types/value.ts

```diff
@@ -235,6 +235,11 @@ export interface IHSVA<T> {
   contents: T[];
 }
 
+export interface INONE<T> {
+  tag: "NONE"
+  contents: T[];
```

Comment:
The `contents` field is not needed here.

Reply:
Removed in updated PR & added downstream logic where needed.